socorro/external/postgresql/base.py

```diff
@@ -87,11 +83,13 @@ def query(self, sql, params=None, error_message=None, connection=None):
                 fresh_connection = True
             # self.context.logger.debug(connection.cursor.mogrify(sql, params))
             results = execute_query_fetchall(connection, sql, params)
+            connection.commit()
```

Comment:
while I converted the middleware to use the current PG ConnectionContext class to make configuration easier, converting the middleware to use the TransactionExecutor was going to be a big job.  Rather than spend the time to do the conversion on code that is already doomed, I just ensured that transactions are at least properly closed.  This is something that the original code should have been doing anyway.